autonomx/Generator.cpp

```diff
@@ -612,8 +612,30 @@ void Generator::applyInputRegion() {
     // iterate over input regions
     for(int i = 0; i < inputRegionSet->rowCount(); i++) {
         GeneratorRegion* region = inputRegionSet->at(i);
+
         int xMax = region->getRect().x() + region->getRect().width();
         int yMax = region->getRect().y() + region->getRect().height();
+
+   /* TODO: create a logic so that the value to input region makes a part of region activate
```

Comment:
Is this TODO still up-to-date?

Reply:
yes. This TODO is not a part of any previous Sprints but came out from a separate discussion with Alex S. I kept it as it is so that it can be later worked upon.